lib/algoliax/resources/object_ecto.ex

```diff
@@ -77,6 +79,8 @@ if Code.ensure_loaded?(Ecto) do
       Algoliax.SettingsStore.delete_settings(tmp_index_name)
       Algoliax.SettingsStore.stop_reindexing(index_name)
 
+      Index.configure_index(module, settings)
```

Comment:
Should not be necessary as `ensure_index` is executed before any other operation.
https://www.algolia.com/doc/api-reference/api-methods/move-index/

> Regarding replicas:
>
> - Moving to an index having replicas is possible, it will replace the destination index data with the source index data, and the data will be propagated to the replicas
> - Moving an index having replicas is not possible, as it would break the relationship with its replicas